src/Middleware/WebSockets/test/UnitTests/HandshakeTests.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace Microsoft.AspNetCore.WebSockets.Tests
+{
+    public class HandshakeTests
+    {
+        [Fact]
+        public void CreatesCorrectResponseKey()
```

Comment:
I just merged the branch from https://github.com/aspnet/AspNetCore/pull/12386 into my branch. That's why this unrelated (to this PR) test is also here.